ethclient/bor_ethclient.go

```diff
@@ -0,0 +1,30 @@
+package ethclient
+
+import (
+	"context"
+
+	ethereum "github.com/maticnetwork/bor"
+	"github.com/maticnetwork/bor/common"
+	"github.com/maticnetwork/bor/core/types"
+)
+
+// GetRootHash returns the merkle root of the block headers
+func (ec *Client) GetRootHash(ctx context.Context, startBlockNumber uint64, endBlockNumber uint64) (string, error) {
+	var rootHash string
+	if err := ec.c.CallContext(ctx, &rootHash, "eth_getRootHash", startBlockNumber, endBlockNumber); err != nil {
+		return "", err
+	}
+	return rootHash, nil
+}
+
+// GetBorBlockReceipt returns bor block receipt
+func (ec *Client) GetBorBlockReceipt(ctx context.Context, hash common.Hash) (*types.BorReceipt, error) {
+	var r *types.BorReceipt
+	err := ec.c.CallContext(ctx, &r, "eth_getBorBlockReceipt", hash)
+	if err == nil {
```

Comment:
Small comment (I'm not a golang dev):

Is not the same as :
```go
if err == nil && r == nil { 
   return nil, ethereum.NotFound
}
```

Reply:
Can we not do it this way ?

```go
var r types.BorReceipt
if err := ec.c.CallContext(ctx, &r, "eth_getBorBlockReceipt", hash); err != nil {
        return nil, err
}

return &r, err
```

Reply:
I will make the change. Thanks.

Reply:
Fixed.